src/main/java/com/adobe/commerce/cif/model/common/Payment.java

```diff
@@ -26,7 +26,7 @@
             "anonymous customer.")
     protected Customer customer;
 
-    @ApiModelProperty(value = "The method for this payment like Card or Cash.", required = true)
+    @ApiModelProperty(value = "The id of the payment method for this payment.", required = true)
     protected String method;
```

Comment:
This looks wired. Shouldn't be PaymentMethod?

Reply:
That would be a backwards incompatible change.

Reply:
Can't we just apply the same treatment as for the payments and have this this deprecated?

Reply:
`paymentMethodId`?

Reply:
+1 for ‘methodId’

Reply:
@cjelger @masodu done.